roles/matrix-client-hydrogen/defaults/main.yml

```
@@ -31,7 +31,14 @@ matrix_client_hydrogen_systemd_required_services_list: ['docker.service']
 matrix_client_hydrogen_self_check_validate_certificates: true
 
 # config.json
+matrix_client_hydrogen_push: {
+  "appId": "io.element.hydrogen.web",
+  "gatewayUrl": "https://matrix.org",
+  "applicationServerKey": "BC-gpSdVHEXhvHSHS0AzzWrQoukv2BE7KzpoPO_FfPacqOo3l1pdqz7rSgmB04pZCWaHPz7XRe6fjLaC-WPDopM"
+}
```

```suggestion
matrix_client_hydrogen_push:
  appId: io.element.hydrogen.web
  gatewayUrl: https://matrix.org
  applicationServerKey: "BC-gpSdVHEXhvHSHS0AzzWrQoukv2BE7KzpoPO_FfPacqOo3l1pdqz7rSgmB04pZCWaHPz7XRe6fjLaC-WPDopM"
```

Isn't it better to keep this more YAML-like?